executor/aggfuncs/aggfuncs.go

```diff
@@ -196,6 +209,15 @@ func (*baseAggFunc) MergePartialResult(sessionctx.Context, PartialResult, Partia
 	return 0, nil
 }
 
+// TODO uncomment it
```

Comment:
Why comment these 2 functions

Reply:
> Why comment these 2 functions

Originally, it's used for marking errors in IDE so that we can know which struct is not implemented. As all structs have been implemented, we have uncommented this.